Sources/CodeEditTextView/CodeEditTextView.swift

```diff
@@ -82,6 +86,7 @@ public struct CodeEditTextView: NSViewControllerRepresentable {
     private var contentInsets: NSEdgeInsets?
     private var isEditable: Bool
     private var letterSpacing: Double
+    private var bracePairHighlight: BracePairHighlight?
```

Comment:
In Xcode, Nova, and VS Code, they use the term "bracket" rather than brace.

<img width="417" alt="Screenshot 2023-05-07 at 7 27 59 PM" src="https://user-images.githubusercontent.com/806104/236710469-c3ba26d1-a657-4436-aa37-5b61bac8d69e.png">

<img width="281" alt="image" src="https://user-images.githubusercontent.com/806104/236710426-657c53a9-12bc-4a36-a774-4b1ba2329fff.png">

<img width="476" alt="image" src="https://user-images.githubusercontent.com/806104/236710500-59885b9e-de07-438f-aae7-2c2c50e81b8e.png">

It is also worth noting that Xcode does use the term "brace" but from what I can tell that is only when it only applies to curly-braces (`{`) and not to other bracket types (`[`, `(`, `<`).

<img width="296" alt="image" src="https://user-images.githubusercontent.com/806104/236710624-10c3f026-b17a-4554-ab29-ef5022a2c9c6.png">

Please change to `private var bracketPairHighlight: BracketPairHighlight?` and change all instances throughout.